packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -1,6 +1,9 @@
-const axios = require(`axios`)
+const got = require(`got`)
```

Comment:
Got automatically retries requests on errors which helps the source plugin recover from occasional glitches